syntax = "proto3";

option go_package = "generated/pb/game";

enum EnterGsType
{
	  LOGIN_NONE = 0;//正常进入，换场景进入
	  LOGIN_FIRST= 1;//第一次登录
	  LOGIN_REPLACE = 2;//顶号
	  LOGIN_RECONNECT = 3;//断线重连
}
	
message PlayerEnterGameStatePbComp
{
	uint32 enter_gs_type = 1;
}

